style/style_script/styles_example.json

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Technically JSON does not allow comments, and several JSON parsers will (correctly) fail the parse when encountered.
Being a sample, it probably doesn't need a copyright header.